src/main/resources/application.conf

```diff
@@ -27,7 +27,7 @@ spray.can.server {
 http {
   interface="0.0.0.0"
   port=8080
-  timeoutSeconds = 1
+  timeoutSeconds = 10
```

Comment:
I don't object to this, but I'm curious .. were you running into any problems? Or just changing this proactively?